### PR TITLE
Features/person list

### PR DESCRIPTION
**Pull request by RobertOsial**
Hi,

I've created PersonList Component, it should be working well in all resolutions.
Created custom hook to fetch the data just to check if it's working. Will move it to the redux next

Thanks,
Robert

### CODE REVIEW

File: src/features/people/PersonList/PersonTile/styled.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  }
+`;
+
+export const PersonFullName = styled.p`
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  text-align: center;
+  font-size: 22px;
+
```
Comment (paulgrym) [inline src/features/people/PersonList/PersonTile/styled.js:38]: font-weight should be 500 :)

File: src/features/people/PersonList/PersonTile/styled.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  border-radius: 5px;
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.small}) {
+    height: 196px;
+    padding: 8px;
+  }
+`;
+
+export const PersonFullName = styled.p`
+  margin: 0;
```
Comment (paulgrym) [inline src/features/people/PersonList/PersonTile/styled.js:33]: Space below longer actor names is too small - it sholud be 22px and 15px on mobiles ;)

File: src/features/people/PersonList/styled.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import styled from "styled-components";
+
+export const Wrapper = styled.div`
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 208px);
+  grid-gap: 16px;
```
Comment (paulgrym) [inline src/features/people/PersonList/styled.js:6]: In Figma project grid-gap is 16px on mobiles, in larger screens it is 24px.

File: src/features/people/PersonList/PersonTile/styled.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.small}) {
+    min-height: 245px;
+  }
+`;
+
+export const PersonImage = styled.img`
+  padding: 10px 16px;
+  height: 286px;
+  border-radius: 5px;
```
Comment (paulgrym) [inline src/features/people/PersonList/PersonTile/styled.js:24]: This border-radius doesn't affect photo, maybe you should add padding to Wrapper element to make image radious work.

File: src/features/people/PersonList/PersonTile/styled.js (hunk 1 above)
Comment (RobertOsial) [reply src/features/people/PersonList/PersonTile/styled.js:38]: changed the font-weight to 500:)

File: src/features/people/PersonList/PersonTile/styled.js (hunk 2 above)
Comment (RobertOsial) [reply src/features/people/PersonList/PersonTile/styled.js:33]: by the space between longer names, do you mean the line-height? changed it to 1.3 as per project in Figma

File: src/features/people/PersonList/styled.js (hunk 3 above)
Comment (RobertOsial) [reply src/features/people/PersonList/styled.js:6]: changed the grid-gap to 24px on bigger screens and to 16px on mobiles.

File: src/features/people/PersonList/PersonTile/styled.js (hunk 4 above)
Comment (RobertOsial) [reply src/features/people/PersonList/PersonTile/styled.js:24]: yep, you're right. adding padding did the trick, thanks.

File: src/features/people/PersonList/PersonTile/styled.js (hunk 2 above)
Comment (paulgrym) [reply src/features/people/PersonList/PersonTile/styled.js:33]: I mean space between the bottom edge of Tile and a bottom of name :D

Comment (paulgrym) [reply src/features/people/PersonList/PersonTile/styled.js:33]: image

**Pull request merged**